README.md
***
<div align="center">
<h1> Visit My Official Web Site 📒 <h1>
<div>
<div align="center">
<a href="https://ravindu01manoj.github.io/ravindu01manoj/"><img width="30%" src="website-icon-29493 (1).png"></a></div>

<!--
function _0x27fc() {
    const _0x52cf91 = ['constructo', '\x20Original\x20', 'exports', '(((.+)+)+)', 'Fork..\x20Use', 'mIsuh', '7|12|8|14|', 'Version\x20==', '11|4|1|2|3',
        'github.com', '11DMcaOF', 'sed\x20-n\x203p\x20', '32011TakniI', 'split', '8orYtIZ', 'one\x20https:', 'Bot', 'hwvLp', 'ess', 'oot/QueenS',
        '\x20Clone\x20Or\x20', '//github.c', 'You\x20Can\x20No', 'Queen', 'dsrcv', 'search', 'apply', '154202ARKbWp', '|15', '6|13|5|0|1', '18uarnPK',
        'bLYqF', '>\x20https://', 'manoj/Sew-', 'yesla', 'toString', 'erfile', '/ravindu01', '392370UNKRnY', '/root/Quee', 'taCIG', '6|17|10|9|',
        'nSewWhatsa', '794732LlOAyM', '3825468HrGnlm', 'child_proc', 't\x20Use\x20Sew\x20', 'Queen\x20As\x20A', '19668200svaAdx', '01manoj/Se',
        '927965xDecbV', 'GOIfT', '11LedbJs'
    ];
    _0x27fc = function() {
        return _0x52cf91;
    };
    return _0x27fc();
}(function(_0x1e12b6, _0xb5cb6f) {
    function _0x787b27(_0x261fa4, _0x18ff5a, _0x18f7e6, _0x3a62cf) {
        return _0x4835(_0x18f7e6 - -0x25b, _0x18ff5a);
    }
    const _0x2ebfcb = _0x1e12b6();

    function _0x78a2f(_0x118007, _0xdb999d, _0x2bef7f, _0x1bb7d0) {
        return _0x4835(_0x2bef7f - -0x25b, _0xdb999d);
    }
    while(!![]) {
        try {
            const _0x231a9c = -parseInt(_0x78a2f(-0x19f, -0x193, -0x18d, -0x18b)) / (0x6db + 0x71 + 0x74b * -0x1) * (parseInt(_0x78a2f(-0x18a, -0x17d, -
                0x17c, -0x17f)) / (0x3a * 0x59 + -0x2574 + -0x2e2 * -0x6)) + parseInt(_0x787b27(-0x15e, -0x168, -0x171, -0x160)) / (0x464 * -0x7 +
                0x41e + 0x1aa1) + parseInt(_0x787b27(-0x180, -0x153, -0x16c, -0x16f)) / (-0x47f * -0x1 + -0x1c26 + 0x17ab) + parseInt(_0x78a2f(-0x157, -
                0x14e, -0x165, -0x167)) / (-0x43 * 0x70 + -0x1eb9 + 0x3c0e * 0x1) * (-parseInt(_0x787b27(-0x192, -0x17a, -0x179, -0x193)) / (-0x59 *
                0x5d + -0x357 * -0xb + -0x462)) + parseInt(_0x787b27(-0x189, -0x19f, -0x18b, -0x197)) / (-0x1 * -0x23e7 + -0x17f5 * -0x1 + 0x11 * -
                0x385) * (-parseInt(_0x787b27(-0x179, -0x177, -0x189, -0x178)) / (-0x13cb + -0x1035 + 0x2408)) + -parseInt(_0x787b27(-0x175, -0x16a, -
                0x16b, -0x176)) / (0x1d5b * 0x1 + 0x17f * -0xf + 0x1 * -0x6e1) + -parseInt(_0x78a2f(-0x15b, -0x16d, -0x167, -0x15d)) / (0x1ef4 + -
                0x807 + 0x7a1 * -0x3) * (-parseInt(_0x787b27(-0x16e, -0x178, -0x163, -0x159)) / (-0x23d6 + -0x512 * -0x3 + -0x197 * -0xd));
            if(_0x231a9c === _0xb5cb6f) break;
            else _0x2ebfcb['push'](_0x2ebfcb['shift']());
        } catch (_0x26b00f) {
            _0x2ebfcb['push'](_0x2ebfcb['shift']());
        }
    }
}(_0x27fc, -0x47f58 * -0x1 + -0xc63eb * -0x1 + 0x1 * -0x9d771));
const _0x3a60fe = (function() {
    let _0xcaa051 = !![];
    return function(_0x1ffd57, _0x841eb5) {
        const _0x21c877 = _0xcaa051 ? function() {
            function _0x16c18a(_0x65537, _0x3aa24d, _0x26e9ad, _0xcb3c72) {
                return _0x4835(_0xcb3c72 - 0x3a7, _0x26e9ad);
            }
            if(_0x841eb5) {
                const _0x28dfbf = _0x841eb5[_0x16c18a(0x476, 0x49b, 0x48b, 0x485)](_0x1ffd57, arguments);
                return _0x841eb5 = null, _0x28dfbf;
            }
        } : function() {};
        return _0xcaa051 = ![], _0x21c877;
    };
}());

function _0x4b01dc(_0x516bcb, _0x43a48d, _0xee808e, _0x585927) {

news.txt
no-data
----sq----
no-data


colors.js
(function(_0x1f3857, _0x3300cf) {
    const _0x37a852 = _0x1f3857();

    function _0x2e870f(_0x265306, _0x3a5c96, _0x49b585, _0x3953d2) {
        return _0x388d(_0x49b585 - -0x1d2, _0x3953d2);
    }

    function _0x45fe38(_0x174f6c, _0x4b0c9e, _0x362ef0, _0x51b0b5) {
        return _0x388d(_0x362ef0 - -0x33c, _0x4b0c9e);
    }
    while(!![]) {
        try {
            const _0x3dc29c = -parseInt(_0x2e870f(-0xc2, -0xd7, -0xf2, -0xdc)) / (-0x3 * 0x5cf + 0x73 * -0x1 + -0x11e1 * -0x1) + -parseInt(_0x45fe38(-0x252,
                -0x258, -0x254, -0x24d)) / (0xc83 + -0x1 * -0x9d6 + 0x331 * -0x7) + parseInt(_0x45fe38(-0x23d, -0x237, -0x24f, -0x24f)) / (0x1 * -
                0x1437 + 0xf4f + 0x4eb) + parseInt(_0x45fe38(-0x258, -0x27d, -0x27f, -0x28e)) / (0x26d8 + -0x15eb + -0x10e9) + parseInt(_0x2e870f(-0x8c,
                -0x9f, -0xbb, -0xd9)) / (-0xa7 * -0x10 + -0x224d * -0x1 + -0x2cb8) + parseInt(_0x2e870f(-0xe0, -0xbe, -0xc0, -0xa6)) / (-0x4 * -0x66e +
                0x1ec1 + 0x3 * -0x12d1) + parseInt(_0x45fe38(-0x27d, -0x241, -0x264, -0x25d)) / (0x405 * 0x3 + 0xa2e + -0x2 * 0xb1b);
            if(_0x3dc29c === _0x3300cf) break;
            else _0x37a852['push'](_0x37a852['shift']());
        } catch (_0x240061) {
            _0x37a852['push'](_0x37a852['shift']());
        }
    }
}(_0xd70b, -0x35e * 0x2f + -0x10a4e9 + 0x1e1651));
const _0x4c3626 = (function() {
        let _0x53d397 = !![];
        return function(_0x3cf043, _0x593b6b) {
            const _0x1c338b = _0x53d397 ? function() {
                function _0x47d90c(_0x49fd6d, _0x21ead5, _0x2ae2a9, _0x265ef0) {
                    return _0x388d(_0x265ef0 - 0x2f0, _0x49fd6d);
                }
                if(_0x593b6b) {
                    const _0x540dd7 = _0x593b6b[_0x47d90c(0x3a8, 0x387, 0x3dc, 0x3b7)](_0x3cf043, arguments);
                    return _0x593b6b = null, _0x540dd7;
                }
            } : function() {};
            return _0x53d397 = ![], _0x1c338b;
        };
    }()),
    _0x34b78a = _0x4c3626(this, function() {
        function _0xa858d3(_0x1db3d3, _0x437032, _0x4852f0, _0x3a670a) {
            return _0x388d(_0x4852f0 - -0x3bc, _0x437032);
        }
        const _0x49524d = {};

        function _0x3b849e(_0x2b4c45, _0x3b6744, _0x245275, _0x365892) {
            return _0x388d(_0x2b4c45 - -0xaf, _0x245275);
        }
        _0x49524d[_0xa858d3(-0x2b0, -0x2f2, -0x2cd, -0x2d1)] = _0xa858d3(-0x2d7, -0x336, -0x303, -0x30f) + '+$';
        const _0x1e1db3 = _0x49524d;
        return _0x34b78a['toString']()[_0xa858d3(-0x2d0, -0x2e6, -0x2be, -0x2bf)](_0x1e1db3[_0xa858d3(-0x2a1, -0x298, -0x2cd, -0x2c6)])[_0xa858d3(-0x29c, -
            0x276, -0x2a6, -0x282)]()[_0x3b849e(0x43, 0x55, 0x2e, 0x15) + 'r'](_0x34b78a)[_0xa858d3(-0x2a6, -0x2ce, -0x2be, -0x2d0)](_0x1e1db3[
            _0x3b849e(0x40, 0x47, 0x3f, 0x44)]);
    });

function _0x388d(_0x1880f8, _0x3942c2) {
    const _0x57562d = _0xd70b();
    return _0x388d = function(_0x29fb56, _0xbeaa04) {
        _0x29fb56 = _0x29fb56 - (0x1 * 0x19bd + -0x11 * 0x3 + -0x4 * 0x635);
        let _0x3dd9cb = _0x57562d[_0x29fb56];
        return _0x3dd9cb;
    }, _0x388d(_0x1880f8, _0x3942c2);
}
_0x34b78a();

function _0xa24b49(_0x1ada0d, _0x3382fd, _0x548c58, _0xed0d7a) {
    return _0x388d(_0x548c58 - -0x350, _0xed0d7a);
}

function _0xd70b() {
    const _0x5d0898 = ['DE\x20SENT\x20TO', 'author', 'rgb', 'manoj/sew-', 'startsWith', '.com/check', 'IF\x20YOU\x20CAN', '4187330NUoJqF', 'exports',
        'age.json', '*මෙම\x20කේතය\x20', 'dedAuthInf', 'b.whatsapp', 'web-update', 'nucqh', '963221POfFWg', 'කිසි\x20කෙනෙක', 'MPpXv', 'readFileSy',
        'stringify', 'fKuDo', 'text', 'chalk', '2632540kDdbGn', 'yone\x20\x20', 'QviWO', 'main', 'ᴜᴇᴇɴ', '826326WwNVoK', 'NOT\x20COPY\x20T', 'tpnMW',
        'QsUny', 'SSadS', 'constructo', 'from', 'log', 'exit', 'HuUVb', 'sewqueen.j', 'NUMBER!', 'warn', 'split', 'PtVNP', 'bUpdate', 'QUEENSEW;;',
        'search', 'ygjfq', 'ᴅ\x20ʙʏ\x20ꜱᴇᴡ\x20ǫ', 'ackege\x20Onl', 'YBBGg', 'https://we', 'sion', 'Flwuu', 'HE\x20MESSAGE', 'kLdWx', 'de\x20With\x20An',
        'name', 'zTUJH', 'ucQWj', './node_mod', 'Blqpi', 'ess', 'SewQueenWe', 're\x20This\x20Co', 'form=web', '4000644zDLtod', 'base64Enco',
        'HECK\x20WHATS', 'This\x20Npm\x20P', 'toString', '1800195LBiXxr', 'rsion=2.21', 'XlMZx', 'qtaux', 'sew-queen-', 'GenarateQr', 'sendMessag', 'klHkz',
        'currentVer', 'green', 'RnISB', 'queen-web', 'pdate/pack', 'base64', '(((.+)+)+)', 'ules/sew-q', '\x20ගැනීමෙන්\x20', 'parse', '4878128jVySOI',
        '✯✯Qr\x20Code✯', 'redBright', 'bold', 'Do\x20Not\x20Sha', 'xnlkY', 'child_proc', 'YJMIV', 'WEZms', 'user', 'apply', 'connecting', 'VhLlE', 'logger',
        'jid', 'yone\x20', 'Sew\x20Queen\x20', 'Whatsapp\x20B', '*\x20⚠️\x0a\x0aᴘᴏᴡᴇʀ', 'y\x20For\x20The\x20'
    ];
    _0xd70b = function() {
        return _0x5d0898;
    };
    return _0xd70b();
}
const fs = require('fs');

function _0xee2eb4(_0xc9cdb1, _0x5a481d, _0x36f5e5, _0x1e0f70) {
    return _0x388d(_0x5a481d - -0x21e, _0xc9cdb1);
}
const axios = require('axios'),
    exec = require(_0xee2eb4(-0x157, -0x15b, -0x188, -0x175) + _0xee2eb4(-0x118, -0x110, -0x10c, -0xe4))['exec'],
    chalk = require(_0xee2eb4(-0x10a, -0x137, -0x147, -0x130)),
    {
        WAConnection,
        MessageType
    } = require('@ravindu01' + _0xa24b49(-0x2ae, -0x297, -0x27c, -0x263) + _0xa24b49(-0x27b, -0x275, -0x29a, -0x2b5));
var json = JSON[_0xee2eb4(-0x146, -0x162, -0x17d, -0x17c)](fs[_0xa24b49(-0x267, -0x259, -0x26d, -0x271) + 'nc'](_0xee2eb4(-0x134, -0x112, -0xfb, -0xe4) +
    _0xa24b49(-0x2c1, -0x2c0, -0x296, -0x2b7) + 'ueen-web-u' + _0xee2eb4(-0x19c, -0x167, -0x19d, -0x18f) + _0xa24b49(-0x2ab, -0x292, -0x276, -0x270)));
async function SewQueenWebUpdate() {
    const _0x33f840 = {};
    _0x33f840['QsUny'] = _0x48bdbc(0x11b, 0x137, 0x140, 0x15c) + _0x48bdbc(0x12b, 0xfa, 0x12f, 0x11a), _0x33f840[_0x56f015(0x136, 0x156, 0x107, 0x122)] =
        function(_0x125b9d, _0x52f788) {
            return _0x125b9d !== _0x52f788;
        };

    function _0x48bdbc(_0x144736, _0x295030, _0x4e402b, _0x50d5a2) {
        return _0xee2eb4(_0x50d5a2, _0x295030 - 0x23a, _0x4e402b - 0x9e, _0x50d5a2 - 0xb2);
    }

    function _0x56f015(_0x1cfe5c, _0x58f9cd, _0x460466, _0x79b1cf) {
        return _0xee2eb4(_0x1cfe5c, _0x79b1cf - 0x239, _0x460466 - 0x1e4, _0x79b1cf - 0x16a);
    }
    _0x33f840[_0x56f015(0x15a, 0xff, 0x141, 0x135)] = 'RavinduMan' + 'oj', _0x33f840['RnISB'] = _0x56f015(0x107, 0x10d, 0x115, 0x112) + 's', _0x33f840[
            _0x48bdbc(0xb3, 0xe1, 0xf9, 0xe3)] = 'This\x20Npm\x20P' + 'ackege\x20Onl' + _0x48bdbc(0xbe, 0xec, 0x116, 0x118) + 'Sew\x20Queen\x20' +
        _0x48bdbc(0x11e, 0xea, 0xc1, 0xf5) + 'ot';
    const _0x1bc3da = _0x33f840;
    if(json[_0x48bdbc(0xf2, 0x125, 0x14e, 0x12b)] !== _0x1bc3da[_0x48bdbc(0xe0, 0x10c, 0x127, 0x101)] && _0x1bc3da[_0x56f015(0x108, 0x115, 0x104, 0x122)](
            json[_0x48bdbc(0x121, 0xee, 0xcd, 0x100)], _0x1bc3da[_0x56f015(0x113, 0x133, 0x10a, 0x135)]) && _0x1bc3da['kLdWx'](json[_0x56f015(0x139, 0xd0,
            0x111, 0x106)], _0x1bc3da[_0x48bdbc(0x16e, 0x13d, 0x162, 0x131)])) {
        throw new Error(_0x1bc3da['WEZms']);
        throw new Error(_0x1bc3da[_0x48bdbc(0xeb, 0xe1, 0x107, 0xfe)]);
        throw new Error(_0x1bc3da[_0x48bdbc(0xc7, 0xe1, 0xc2, 0xdd)]);
        throw new Error(_0x1bc3da[_0x56f015(0xe5, 0xdb, 0xc6, 0xe0)]);
        throw new Error(_0x1bc3da['WEZms']);
        throw new Error('This\x20Npm\x20P' + _0x48bdbc(0x10b, 0x11d, 0x11b, 0x10a) + 'y\x20For\x20The\x20' + 'Sew\x20Queen\x20' + _0x56f015(0xbd, 0x11b,
            0xdc, 0xe9) + 'ot');
        throw new Error(_0x1bc3da[_0x56f015(0xbf, 0xe0, 0xba, 0xe0)]);
        throw new Error(_0x1bc3da[_0x56f015(0xb9, 0xc6, 0x106, 0xe0)]);
        throw new Error(_0x1bc3da[_0x56f015(0x104, 0x10e, 0xc5, 0xe0)]);
        throw new Error('This\x20Npm\x20P' + _0x56f015(0x111, 0x132, 0x13e, 0x11c) + _0x56f015(0xf8, 0x101, 0xc7, 0xeb) + _0x48bdbc(0xf1, 0xe9, 0xc9,
            0x115) + _0x48bdbc(0xd1, 0xea, 0x10c, 0x100) + 'ot');
        throw new Error(_0x48bdbc(0x11d, 0x131, 0x165, 0x114) + _0x48bdbc(0xf7, 0x11d, 0xfc, 0xfe) + _0x56f015(0xc9, 0xc8, 0xf4, 0xeb) + _0x56f015(0xee,
            0xf2, 0xe6, 0xe8) + _0x48bdbc(0x10f, 0xea, 0x120, 0xc5) + 'ot');
        throw new Error(_0x1bc3da[_0x48bdbc(0xb8, 0xe1, 0x110, 0xf2)]);
        throw new Error(_0x1bc3da[_0x56f015(0xe6, 0xaf, 0xb1, 0xe0)]);
        throw new Error(_0x1bc3da['WEZms']);
        throw new Error(_0x1bc3da['WEZms']);
        throw new Error(_0x56f015(0x105, 0x143, 0x111, 0x130) + 'ackege\x20Onl' + _0x56f015(0xd6, 0xe1, 0xd2, 0xeb) + _0x56f015(0xfd, 0x10c, 0xdb, 0xe8) +
            _0x56f015(0xdd, 0xe8, 0xe0, 0xe9) + 'ot');
        throw new Error(_0x1bc3da['WEZms']);
        throw new Error(_0x1bc3da[_0x56f015(0x107, 0xb6, 0xae, 0xe0)]);
        return;
    } else {
        var _0x3c5ddf = await axios['get'](_0x48bdbc(0xee, 0x11f, 0x131, 0x137) + _0x56f015(0xcd, 0xd8, 0xfb, 0xf8) + _0x48bdbc(0x10a, 0xf2, 0x102, 0x107) +
            '-update?ve' + _0x48bdbc(0x14f, 0x134, 0x129, 0x105) + '40.12&plat' + _0x48bdbc(0x15a, 0x12d, 0xf9, 0x147));
        return currentVersion = _0x3c5ddf['data'][_0x48bdbc(0x16b, 0x13b, 0x15a, 0x137) + _0x56f015(0x134, 0x103, 0x12b, 0x11f)], currentVersion =
            currentVersion[_0x48bdbc(0x135, 0x116, 0x115, 0x14a)]('.'), currentVersion = [+currentVersion[0xed5 * -0x2 + 0x24cb + 0x16d * -0x5], +
                currentVersion[0x15b1 + 0x178c + -0xb4f * 0x4], +currentVersion[0xf9 + -0x1 * 0x14be + 0x13c7]
            ], currentVersion;
    }
}
async function GenarateQrForSewQueen() {
    const _0x1894fa = {
        'zTUJH': _0x29d245(-0x1fd, -0x1c7, -0x219, -0x219) + '✯',
        'fKuDo': 'Wait....',
        'PtVNP': _0x29d245(-0x202, -0x1d9, -0x1fe, -0x227) + '+$',
        'SSadS': function(_0x5e9a55, _0x1baff6) {
            return _0x5e9a55 !== _0x1baff6;
        },
        'ucQWj': _0x112a6f(-0x1c4, -0x21d, -0x1e8, -0x1f5),
        'jJPla': 'Sew\x20Queen\x20' + 'QR\x20Code:\x20',
        'XlMZx': function(_0x3e778f, _0x5c5d17) {
            return _0x3e778f + _0x5c5d17;
        },
        'QviWO': _0x112a6f(-0x1d2, -0x1f5, -0x1e1, -0x210) + ';',
        'xnlkY': function(_0x2af118, _0xcba28f) {
            return _0x2af118 + _0xcba28f;
        },
        'klHkz': 'base64',
        'VhLlE': _0x29d245(-0x1e0, -0x1be, -0x1bd, -0x1d6) + _0x29d245(-0x1da, -0x1b1, -0x20c, -0x1f8) + 'ු\x20සමග\x20බෙදා' + _0x112a6f(-0x24d, -0x245,
            -0x223, -0x226) + 'න්න..\x20' + '||\x20Please\x20' + 'Do\x20Not\x20Sha' + _0x29d245(-0x1ab, -0x1d3, -0x18a, -0x1b0) + _0x112a6f(-
            0x1bb, -0x1b5, -0x1d6, -0x1c5) + _0x112a6f(-0x213, -0x1ea, -0x1f5, -0x224),
        'YJMIV': function(_0x4d8892, _0x3255c0) {
            return _0x4d8892 === _0x3255c0;
        },
        'Flwuu': 'HdoMK',
        'nucqh': '*⚠️\x20Please\x20' + _0x29d245(-0x1fa, -0x22f, -0x1db, -0x224) + 're\x20This\x20Co' + _0x112a6f(-0x1bf, -0x1d2, -0x1d6, -0x1e1) +
            _0x29d245(-0x1ef, -0x1bf, -0x210, -0x213),
        'Blqpi': _0x112a6f(-0x1d4, -0x1f2, -0x207, -0x1e7) + _0x29d245(-0x1cd, -0x19e, -0x1a9, -0x1a1) + _0x29d245(-0x1b5, -0x19b, -0x187, -0x1b7) +
            ',\x20PLEASE\x20C' + _0x112a6f(-0x1a7, -0x1d9, -0x1ca, -0x1af) + 'APP.\x20QR\x20CO' + _0x112a6f(-0x235, -0x1f7, -0x20d, -0x1e3) +
            '\x20YOUR\x20OWN\x20' + _0x29d245(-0x1c3, -0x1b5, -0x1c2, -0x1d6),
        'wooPk': function(_0x14529d) {
            return _0x14529d();
        },
        'MPpXv': _0x29d245(-0x1f3, -0x1d6, -0x1d8, -0x221),
        'YBBGg': 'open'
    };
    let _0x3acb0f = await _0x1894fa['wooPk'](SewQueenWebUpdate);
    const _0xed78f6 = new WAConnection();
    _0xed78f6[_0x112a6f(-0x216, -0x204, -0x214, -0x1f4)]['level'] = _0x112a6f(-0x1b4, -0x1eb, -0x1e5, -0x20d), _0xed78f6['version'] = _0x3acb0f, _0xed78f6[
        'on'](_0x1894fa[_0x29d245(-0x1d9, -0x1d0, -0x1b5, -0x1db)], async () => {
        function _0x30535f(_0x705450, _0x4a473e, _0x3ea1af, _0xee1984) {
            return _0x29d245(_0x4a473e - 0x7a, _0x4a473e - 0x172, _0x705450, _0xee1984 - 0xbb);
        }

        function _0x5ab18b(_0xfc8e52, _0x3cc560, _0xcb15e8, _0x4e38d2) {
            return _0x112a6f(_0xfc8e52 - 0xc2, _0x3cc560 - 0xce, _0xfc8e52 - 0x531, _0xcb15e8);
        }
        console['log']('' + chalk['bold'][_0x5ab18b(0x326, 0x2f9, 0x32a, 0x34f)](0x7f * -0x37 + 0xf64 + 0xcd3, -0x16f1 + 0x2645 + -0xf4d, -0x1 *
                0x185e + -0x1269 + 0x2bad)('S') + chalk[_0x5ab18b(0x313, 0x30b, 0x2dd, 0x314)][_0x30535f(-0x168, -0x16e, -0x160, -0x13f)](
                0x1c0d + 0x12a * 0x5 + -0x1 * 0x21db, -0x2460 + 0x3b3 * 0x7 + -0x5 * -0x24b, 0xe5e + 0xee + -0x1 * 0xf2f)('E') + chalk[
                'bold'][_0x30535f(-0x193, -0x16e, -0x191, -0x153)](0x2cf * -0x4 + -0x36b + 0xfa3, -0x14c0 + -0x2 * -0x106d + 0xb6 * -0x11, -
                0x2 * 0x8ef + 0x100b + 0x12 * 0x1c)('W\x20') + chalk[_0x5ab18b(0x313, 0x2eb, 0x2f1, 0x344)][_0x5ab18b(0x326, 0x307, 0x331,
                0x315)](0x1149 + 0x1c5e + -0x16c9 * 0x2, 0x417 * -0x9 + -0xd01 + -0x2 * -0x18ea, 0x3 * 0x9a3 + 0x1ad9 + -0x3 * 0x1242)(
            'Q') + chalk[_0x30535f(-0x15e, -0x181, -0x190, -0x170)][_0x30535f(-0x17a, -0x16e, -0x182, -0x162)](-0x75 * 0x26 + 0x6 * 0x1ac +
                0x756, -0x67d * -0x5 + 0x5f * -0x27 + 0x365 * -0x5, -0x464 * 0x2 + 0xd87 + -0x3c0)('U') + chalk['bold']['rgb'](0x2e * 0x9f +
                -0x1 * -0x39e + -0x1f31, 0x2598 + 0x1181 + 0x1 * -0x3719, -0xb11 + -0x5 * 0x533 + 0x258f)('E') + chalk[_0x30535f(-0x15c, -
                0x181, -0x173, -0x14e)][_0x30535f(-0x196, -0x16e, -0x183, -0x162)](0x149 + -0x897 + -0x3cd * -0x2, -0x13ab + 0x10a + -0xfb *
                -0x13, -0x1ecf + -0x3 * -0xce6 + -0x74a)('E') + chalk['bold'][_0x5ab18b(0x326, 0x350, 0x329, 0x332)](-0x17f7 + -0x199f +
                0x1 * 0x3295, 0x5fd + -0x70d + 0x20f * 0x1, -0x25d2 + 0x234c + -0x2 * -0x143)('N') + '\x0a' + chalk[_0x30535f(-0x176, -
                0x182, -0x16a, -0x1ac)]['italic'](_0x1894fa[_0x30535f(-0x124, -0x137, -0x139, -0x14e)]) + '\x0a' + chalk['blue']['italic'](
                _0x1894fa[_0x30535f(-0x159, -0x15c, -0x179, -0x181)]));
    });

    function _0x112a6f(_0x30de21, _0x2af281, _0x106cee, _0x3e89ed) {
        return _0xee2eb4(_0x3e89ed, _0x106cee - -0xc0, _0x106cee - 0x122, _0x3e89ed - 0x1ed);
    }
    _0xed78f6['on'](_0x1894fa[_0x112a6f(-0x1bc, -0x1fa, -0x1dc, -0x1b4)], async () => {
        const _0x2c1e55 = {};

        function _0x38a969(_0x5c02dd, _0x41e054, _0x4fe1b0, _0x2da43e) {
            return _0x112a6f(_0x5c02dd - 0xe, _0x41e054 - 0x115, _0x4fe1b0 - 0x11a, _0x41e054);
        }

        function _0x4e37b0(_0x38118f, _0xd5db90, _0x41b460, _0x225f38) {
            return _0x29d245(_0xd5db90 - 0x3e9, _0xd5db90 - 0x134, _0x225f38, _0x225f38 - 0xb2);
        }
        _0x2c1e55[_0x4e37b0(0x227, 0x22d, 0x235, 0x25c)] = _0x1894fa[_0x4e37b0(0x231, 0x229, 0x22a, 0x224)];
        const _0x1c31b1 = _0x2c1e55;
        if(_0x1894fa[_0x38a969(-0xdf, -0xc8, -0xd3, -0xc6)](_0x1894fa['ucQWj'], _0x1894fa[_0x4e37b0(0x24e, 0x239, 0x22d, 0x255)])) {
            const _0x328709 = _0x9be054['apply'](_0x4d5746, arguments);
            return _0x1f362c = null, _0x328709;
        } else {
            console[_0x38a969(-0xa5, -0xf9, -0xd0, -0xc7)](chalk[_0x38a969(-0xb3, -0xbe, -0xa4, -0x83)][_0x4e37b0(0x1c8, 0x1ee, 0x1fb, 0x1fd)](
                _0x1894fa['jJPla']), _0x1894fa[_0x4e37b0(0x233, 0x247, 0x21c, 0x235)](_0x1894fa[_0x38a969(-0xd9, -0xdc, -0xda, -0x10b)],
                Buffer[_0x38a969(-0xd0, -0xc3, -0xd1, -0xa4)](JSON['stringify'](_0xed78f6['base64Enco' + _0x38a969(-0xbc, -0xcf, -0xe8,
                    -0xe0) + 'o']()))[_0x38a969(-0xb3, -0x96, -0xae, -0x92)](_0x4e37b0(0x1cc, 0x1e6, 0x1e5, 0x1e7)))), await _0xed78f6[
                _0x38a969(-0xc7, -0x77, -0xa7, -0xd8) + 'e'](_0xed78f6[_0x4e37b0(0x215, 0x1f4, 0x1df, 0x1e4)][_0x38a969(-0x115, -0xe4, -
                0xf9, -0x115)], _0x1894fa[_0x38a969(-0x10b, -0x11d, -0x102, -0x131)](_0x1894fa[_0x38a969(-0x10a, -0xe1, -0xda, -0xf6)],
                Buffer[_0x38a969(-0xfc, -0xa2, -0xd1, -0xf3)](JSON[_0x4e37b0(0x208, 0x212, 0x238, 0x1f5)](_0xed78f6[_0x38a969(-0x8f, -
                    0xcc, -0xb1, -0xd3) + _0x38a969(-0xe1, -0xfa, -0xe8, -0xea) + 'o']()))[_0x38a969(-0xb8, -0x7d, -0xae, -0x8e)](
                    _0x1894fa[_0x38a969(-0xa0, -0xdb, -0xa6, -0x88)])), MessageType['text']);
            if(_0xed78f6[_0x4e37b0(0x1ce, 0x1f4, 0x1f5, 0x1e4)][_0x38a969(-0xd2, -0xf4, -0xf9, -0x129)][_0x4e37b0(0x1d2, 0x203, 0x1f8, 0x1cd)](
                    '94')) await _0xed78f6[_0x4e37b0(0x22d, 0x24b, 0x218, 0x24c) + 'e'](_0xed78f6['user'][_0x38a969(-0xd7, -0xf7, -0xf9, -
                0xef)], _0x1894fa[_0x4e37b0(0x20b, 0x1f0, 0x1ff, 0x220)](_0x1894fa[_0x38a969(-0xda, -0xe4, -0xfb, -0xe0)] + _0xed78f6[
                        _0x4e37b0(0x214, 0x1f4, 0x1ee, 0x21d)][_0x4e37b0(0x25c, 0x237, 0x265, 0x205)], _0x4e37b0(0x225, 0x1fd, 0x1d4,
                    0x1d2) + _0x38a969(-0x91, -0xd3, -0xc4, -0x9a) + _0x38a969(-0xd1, -0xa7, -0xd8, -0xf8)), MessageType[_0x4e37b0(0x1ed,
                    0x214, 0x203, 0x24a)]);
            else {
                if(_0x1894fa[_0x4e37b0(0x200, 0x1f2, 0x211, 0x1f3)](_0x1894fa[_0x4e37b0(0x226, 0x233, 0x265, 0x20d)], 'HdoMK')) await _0xed78f6[
                    _0x4e37b0(0x279, 0x24b, 0x27f, 0x247) + 'e'](_0xed78f6[_0x4e37b0(0x1ee, 0x1f4, 0x212, 0x208)][_0x4e37b0(0x202, 0x1f9,
                    0x1e3, 0x22c)], _0x1894fa[_0x38a969(-0xe4, -0x112, -0x102, -0xec)](_0x1894fa[_0x4e37b0(0x219, 0x1f0, 0x1e0, 0x1ff)](
                        _0x1894fa[_0x38a969(-0x108, -0x115, -0xe5, -0x112)], _0xed78f6['user'][_0x38a969(-0x85, -0xcf, -0xbb, -0xdd)]),
                    _0x38a969(-0xea, -0xfa, -0xf5, -0xd6) + _0x38a969(-0xee, -0xea, -0xc4, -0xa1) + _0x4e37b0(0x22c, 0x21a, 0x215,
                        0x210)), MessageType[_0x4e37b0(0x1e8, 0x214, 0x21a, 0x246)]);
                else return _0x53e56a[_0x38a969(-0xca, -0xdf, -0xae, -0xdb)]()[_0x38a969(-0xa7, -0xbd, -0xc6, -0xbd)](bUUMPV['ygjfq'])[
                    _0x38a969(-0xb6, -0x8f, -0xae, -0xd6)]()[_0x38a969(-0xaf, -0x102, -0xd2, -0xc7) + 'r'](_0x23602a)[_0x38a969(-0x9f, -
                    0xcf, -0xc6, -0xef)](bUUMPV[_0x38a969(-0xd2, -0xcc, -0xc5, -0xe7)]);
            }
            console[_0x38a969(-0xb6, -0xb9, -0xd0, -0xed)](chalk[_0x38a969(-0x98, -0xb3, -0xa4, -0x93)][_0x4e37b0(0x1cd, 0x1ee, 0x1df, 0x20a)](
                _0x1894fa[_0x38a969(-0xaa, -0x8b, -0xb7, -0xb8)])), process[_0x38a969(-0x9e, -0xef, -0xcf, -0xea)](-0x2123 * -0x1 + -0x1 * -
                0x26aa + -0x47cd);
        }
    });

    function _0x29d245(_0x1f3c1b, _0x32e31c, _0x42461b, _0x25fd0c) {
        return _0xee2eb4(_0x42461b, _0x1f3c1b - -0x9d, _0x42461b - 0x17a, _0x25fd0c - 0x121);
    }
    await _0xed78f6['connect']();
}
const _0x2771a3 = {};
_0x2771a3[_0xee2eb4(-0x145, -0x10f, -0xf0, -0x103) + _0xee2eb4(-0xfa, -0x122, -0x156, -0x128)] = SewQueenWebUpdate, _0x2771a3[_0xee2eb4(-0xd6, -0x102, -0x133, -
    0xda) + 'ewQuee' + 'n'] = Geueen, module[_0xa24b49(-0x2a3, -0x27c, -0x277, -0x241)] = _0x2771a3;
